spec/controllers/admin/cafe_works_controller_spec.rb

```diff
@@ -0,0 +1,146 @@
+require 'rails_helper'
+
+RSpec.describe Admin::CafeWorksController, type: :controller do
+  let(:user) { create(:user) }
+  let(:not_owner) { create(:user) }
+  let(:cwork_profile) { create(:cafe_work, :w_profile, profile: user.profile) }
+  let(:cwork_access) { create(:cafe_work, :access) }
+  let(:cwork) { create(:cafe_work) }
+
+  # Hack to become admin
+  # Should be changed when Cancancan is implemented
+  # /d.wessman
+  before(:each) do
+    sign_in user
```

Comment:
Try to replace this with `allow(controller).to receive(:currrent_user).and_return(user)`